doc/xml/phrases-decl.ent

```diff
@@ -161,27 +161,16 @@
 <command>zypper</command> command.</para>
     <procedure>
      <title>Installing &dapsacr; via Zypper From &suse-dapsrepo;</title>
-     <step>
-      <para>Open a browser and enter the following URL: <ulink url="http://download.opensuse.org/repositories/Documentation:/Tools"></ulink></para>
-     </step>
-     <step>
-      <para>Select your distribution and product number to make the browser show the URL for the respective repository.</para>
-     </step>
-     <step>
-      <para>Copy the URL from the address bar.</para>
-     </step>
      <step>
       <para>Open a terminal.</para>
      </step>
      <step>
-      <para>Add the repository with the following zypper command:</para>
-      <screen>&prompt.root;<command>zypper ar -f <replaceable>URL</replaceable> &suse-dapsrepo;</command></screen>
-      <para>Replace <replaceable>URL</replaceable> with the URL you pasted from
-      your browser.</para>
+      <para>Add the &suse-dapsrepo; repository with the following zypper command:</para>
+      <screen>&prompt.root;<command>zypper addrepo --refresh obs://Documentation:Tool &suse-dapsrepo;</command></screen>
```

Comment:
1. There is an s missing:
   ```suggestion
         <screen>&prompt.root;<command>zypper addrepo --refresh obs://Documentation:Tools &suse-dapsrepo;</command></screen>
   ```

2. `&suse-dapsrepo;` = `Documentations:Tools`, meaning you can just use that entity twice in the command (or use something shorter for the repo alias, like `doc-tools`)

Reply:
1. Ahh, right! My bad, thanks for spotting this. :+1: 
2. Yes, that would be better for. I've added it in commit 21820e7d